Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-10-04
Initial Package Version: 0.8.0
Upstream Status:         Committed to upstream CVS
Origin:                  Upstream CVS
                         http://cvs.sourceforge.net/viewcvs.py/libofx/libofx/lib/
Description:             Fixes GCC-4.0.x build problem


diff -Naur libofx-0.8.0-orig/lib/tree.hh libofx-0.8.0/lib/tree.hh
--- libofx-0.8.0-orig/lib/tree.hh	2004-04-09 01:51:45.000000000 -0500
+++ libofx-0.8.0/lib/tree.hh	2005-10-04 16:26:41.000000000 -0500
@@ -69,7 +69,7 @@
    }
 
 template <class T1>
-inline void kp::destructor(T1* p)
+inline void destructor(T1* p)
    {
    p->~T1();
    }
